tests/gold_tests/pluginTest/test_hooks/body_buffer.test.py

```diff
@@ -99,10 +99,10 @@ def setupTS(self):
         })
 
         self._ts.Streams.stderr = Testers.ContainsExpression(
-            rf"request_buffer_plugin gets the request body with length\[{self.content_length_size}\]",
+            rfr"request_buffer_plugin gets the request body with length\[{self.content_length_size}\]",
```

Comment:
'make autopep8' made the changes in this file.

Reply:
I saw you reverted this part of the change. Is autopep8 making invalid code changes/suggestions?

Reply:
Yes, the autest failed with the rfr changes.  Perhaps my local "make autopep8" is messed up.

Reply:
@serrislew saw this too with PR https://github.com/apache/trafficserver/pull/8248 Oh, and I guess the same file. at least autopep8 is consistent!

Reply:
I changed `rf` to `fr` which satisfied autopep8 and the autest